### ISSUE TITLE
Can't find hash.k2d, taxo.k2d and opts.k2d after kraken2 --build, but only taxo.k2d.tmp

### ISSUE THREAD

**Issue by MetaDan89** [DerrickWood/kraken2#279]
Hello,

I am attempting to build a custom (nr) kraken2 database, but I can't find the file in directory.

Using these commands:

```
./kraken2-build --download-taxonomy --db /data2/d.pietrucci_2/Kraken_db
nohup ./kraken2-build --download-library nt --threads 8 --no-masking --db /data2/d.pietrucci_2/Kraken_db/ &
nohup ./kraken2-build --build --db /data2/d.pietrucci_2/Kraken_db/ &
```

I got this output:

```
Downloading nt database from server... done.
Uncompressing nt database...done.
Parsing nt FASTA file...done.
Creating sequence ID to taxonomy ID map (step 1)...
Found 65915203/65915672 targets, searched through 753852034 accession IDs, search complete.
lookup_accession_numbers: 469/65915672 accession numbers remain unmapped, see unmapped.txt in DB directory
Sequence ID to taxonomy ID map complete. [17m45.218s]
Estimating required capacity (step 2)...
Estimated hash table requirement: 197379054444 bytes
Capacity estimation complete. [6h31m52.395s]
Building database files (step 3)...
Taxonomy parsed and converted.
CHT created with 21 bits reserved for taxid.

```

Then, in the Kraken_db directory, I can find only one file (taxo.k2d.tmp). What I should do to obtain the hash.k2d, taxo.k2d and opts.k2d file?

**Comment by jenniferlu717** [comment]
The build command line may not have finished. You should likely run the `kraken2-build --build` command again using multiple threads. The nt database is fairly large and building can take awhile especially if done single-threaded.

**Comment by MetaDan89** [comment]
Thank you very much!
It's possibile to restart the building after the step 2 (Capacity estimation complete)?

**Comment by jenniferlu717** [comment]
You can restart the build, but since the files were not generated, it will likely repeat step 2 again (not step 1 though)

**Comment by jenniferlu717** [comment]
As its been a few months, I'm going to close this issue for now. If you continue to have problems, please open a new issue.

**Comment by PeterCx** [comment]
Hi there,

I am having a similar problem. I am trying to build the nt database in Kraken2 using the following command:

kraken2-build --build --threads 30 --db DIRECTORY

But when it finishes in my db folder I have just 3 files; seqid2taxid.map and taxo.k2d.tmp and unmapped.txt 

As mentioned above I have restarted the build using the same command twice now (so 3rd time trying to build). 

I understand this is a large database but I started the original build last Tuesday. Memory or space is not an issue. 

Many thanks in advance.
